.changeset/sweet-badgers-reflect.md

```diff
@@ -0,0 +1,394 @@
+---
+'@backstage/config-loader': minor
+---
+
+Added provision so that $file and $include can use a url (i.e. if the value starts with http:// or https://)
+
+These changes are **required** to packages/config-loader/src/lib/index.ts
+
+```diff
```

Comment:
I think these Markdown changes aren't necessary.  A changeset describes what you've changed. But the code changes themselves are IN the code. Once this is released, a user would make use of them by bumping their packages to use the new `@backstage/config-loader`; they would not follow these instructions to modify code in their instance since they likely would not have this code in their instance?  So think this should just explain to the user if there's anything they need to DO to use this?

Reply:
Hi @adamdmharvey
Thanks for taking time to review and your responses.

Sure.  Sorry, I have realised my mistake now. I mistook the documentation to include all the changes. I will be removing them.